ToDoApp/ToDoApp/ToDoListViewModel.swift

```diff
@@ -37,14 +37,17 @@ final class ToDoListViewModel: ObservableObject {
         if case .loaded = state {
             return
         }
+        loading = true
         fetchToDoTasks()
     }
     // Handle add button tapped
     func addButtonTapped() {
+        loading = true
         addEditButtonTapped(mode: .createNewTask)
     }
     // Handle edit button tapped
     func editButtonTapped(todo: TaskModel) {
+        loading = true
```

Comment:
why would we spread assigning true to loading across the code? there is no loading happening when handling add and edit buttons are tapped

Reply:
Without having loading, when edit or add a task, the screen is not updated

https://user-images.githubusercontent.com/88293742/208746449-d7e04ff6-284f-4d49-809d-ae6dbc6a0316.mp4

Reply:
What i mean is that true can be simply assigned only in the fetch method